k8s/contiv-vpp/values.yaml

```diff
@@ -1,6 +1,7 @@
 ---
 contiv:
-  useNoOverlay: false
+  useNoOverlay: true
```

Comment:
same as above, please do not change default settings.

Reply:
the same as above (see https://github.com/contiv/vpp/pull/1523/commits/89026877015c3b75767871284bc6960e939eac69)